puppetserver-standalone/spec/dockerfile_spec.rb

```diff
@@ -24,6 +24,15 @@
     it { should be_executable }
   end
 
+  describe package('r10k') do
+    it { is_expected.to be_installed }
```

Comment:
This fails in travis because package assumes a system package, where as r10k is a gem. You can look for system gems with `by('gem')` see http://serverspec.org/resource_types.html#package

But that won't I don't think check the Puppet ruby path. You might find a suitable matcher but otherwise I would remove this. The presence of the binary is a useful check on it's own I think.